templates/web/base/report/_inspect.html

```diff
@@ -69,6 +80,15 @@ <h2 class="inspect-form-heading">[% loc('Inspect report') %]</h2>
           [% INCLUDE 'report/inspect/public_update.html' %]
         [% END %]
 
+        [% IF problem.get_extra_metadata('inspected') %]
```

Comment:
Could we get this in Oxfordshire template only? Just worried that it might confuse current users (in case they e.g. have "all permissions") to see this when nothing would be happening for them.